add-on/chrome/test/mochitest/browser_panel_privateBrowsing.js

```diff
@@ -0,0 +1,26 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
+"use strict";
+
+var win;
+
+add_task(function* test_panel_should_be_shown() {
+  win = yield BrowserTestUtils.openNewBrowserWindow();
+  yield win.LoopUI.openPanel();
+
+  Assert.notEqual(document.getElementById("loop-notification-panel"), null, "Panel exists");
+});
+
+add_task(function* test_init_copy_panel_private() {
+  win = yield BrowserTestUtils.openNewBrowserWindow({ private: true });
+  yield win.LoopUI.openPanel();
+
+  Assert.equal(win.document.getElementById("loop-notification-panel"), null, "Panel doesn't exist for private browsing");
+});
+
+registerCleanupFunction(function* () {
+  if (win) {
+    yield BrowserTestUtils.closeWindow(win);
+  }
+});
```

Comment:
@Standard8 I don't really know what I'm doing wrong. Both tests seems to work but when I run them, an extra window is being opening. Any ideas?